kpatch-build/create-diff-object.c

```diff
@@ -3082,8 +3082,10 @@ int main(int argc, char *argv[])
 			break;
 		}
 	}
-	if (!hint)
-		ERROR("FILE symbol not found in base. Stripped?\n");
+	if (!hint) {
+		log_normal("WARNING: FILE symbol not found in base. Stripped object file or assembly source?\n");
+		return 3; /* NO_CHANGE */
```

Comment:
nit: not about this particular change, but we might want to add an enum for returncodes, it is inconvenient to search for those in comments.

Reply:
I thought the same while modifying the code, but for some reason assumed it was a stylistic choice that @jpoimboe made when he introduced the numbers.  If I remember correctly, the return codes are also used by one of the fatal error message macros, so the enum would need to go into a common header file.  There's also stdlib.h's notion of EXIT_SUCCESS and EXIT_FAILURE ... is there a safe way to use those and add a EXIT_NO_CHANGE?

Reply:
It is used in DIFF_FATAL macros from create-diff-object.c (retval 2) and in ERROR macros from log.h (retval1).
I think using stdlib's defines alongside our own might be confusing.